packages/frontend/app/register/components/get-weekly-meetings.tsx

```diff
@@ -1,22 +1,22 @@
 "use client";
 
-import { fetchLocalMeetings } from "@/lib/find-meetings/fetch-meetings";
-import { Congregation } from "@/lib/types/congregation";
 import React from "react";
 import { WeeklyMeetingsList } from "./weekly-meetings-list";
 import { Button } from "@nextui-org/button";
 import { MoveRight } from "lucide-react";
 import { createCongregation } from "@/lib/find-meetings/create-congregation";
 import { useRouter } from "next/navigation";
+import { fetchLocalMeetingsThunk } from "@/lib/stores/local-meetings";
```

Comment:
Thunk???

Reply:
A thunk is a CS concept, basically a routine that is delayed, then its result is injected into where it's needed. Same principle here. 

Weird name ik.